addons/l10n_fr_account/i18n/l10n_fr_account.pot

```diff
@@ -305,6 +324,12 @@ msgstr ""
 msgid "PieceRef"
 msgstr ""
 
+#. module: l10n_fr_account
+#. odoo-python
+#: code:addons/l10n_fr_account/models/res_partner.py:0
+msgid "SIRET Number"
+msgstr ""
```

Comment:
While you're adding it, translate it :) (Numéro Siret)
I know the rest isn't. It should but :shrug: